ddl/ddl_api.go

```diff
@@ -2305,6 +2338,75 @@ func (d *ddl) AddColumn(ctx sessionctx.Context, ti ast.Ident, spec *ast.AlterTab
 	return errors.Trace(err)
 }
 
+// AddColumns will add multi new columns to the table.
+func (d *ddl) AddColumns(ctx sessionctx.Context, ti ast.Ident, specs []*ast.AlterTableSpec) error {
+	schema, t, err := d.getSchemaAndTableByIdent(ctx, ti)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	var columns []*table.Column
+	var positions []*ast.ColumnPosition
+	var offsets []int
+
+	// Check all the columns at once.
+	newColumnsCount := 0
+	addingColumnNames := make(map[string]bool)
+	for _, spec := range specs {
+		for _, specNewColumn := range spec.NewColumns {
+			newColumnsCount++
+			if !addingColumnNames[specNewColumn.Name.Name.O] {
```

Comment:
Use `specNewColumn.Name.Name.L` instead of `specNewColumn.Name.Name.O`.

Currently:
```sql
l>alter table t add column c int, add column C int;
(8214, u'Cancelled DDL job')  
```
The error message was not right.